packages/styleUtils/layout/gridList/components/GridList.tsx

```diff
@@ -8,46 +8,62 @@ import {
 } from "../../../../shared/styles/styleUtils";
 import { BreakpointConfig } from "../../../../shared/styles/breakpoints";
 
-interface GridListProps {
+export interface GridListProps {
   /**
-   * How many columns the grid should be. Can be set for all viewport sizes, or configured to have different values at different viewport width breakpoints
+   * Sets the amount of grid columns explicitly. Can be set for all viewport sizes, or configured to have different values at different viewport width breakpoints. If not set, automatic responsive grid styling will be in effect.
    */
-  columnCount: BreakpointConfig<number>;
+  columnCount?: BreakpointConfig<number>;
   /**
-   * The size of the space between each list item. Can be set for all viewport sizes, or configured to have different values at different viewport width breakpoints
+   * The size of the space (gap) between each list item. Can be set for all viewport sizes, or configured to have different values at different viewport width breakpoints. The default size is set to medium ("m" = 16px);
    */
   gutterSize?: SpaceSize;
   /**
-   * Which HTML list tag to render the component with
+   * Which HTML list tag to render the component with.
    */
-  tag: "ul" | "ol";
+  tag?: "ul" | "ol";
   children:
     | Array<React.ReactElement<HTMLLIElement>>
     | React.ReactElement<HTMLLIElement>;
+  /**
+   * Additional styles to extend the component.
+   */
+
+  className?: string;
+
+  /**
+   * Human-readable selector used for writing tests.
+   */
+  ["data-cy"]?: string;
+  /**
+   * Centers grid children by implementing `place-items: center;` Optimal for grids that could have a state with only 1 grid child.
+   */
+  centerItems?: boolean;
```

Comment:
need a bit to chew on this. There's a vertical and a horizontal centering to consider, and this one property would by default set both. it is pretty different than the current default behavior.

Reply:
🤔 By default, this will not be true. I was hoping to simplify centering by using place-items as a one-liner to toggle on or off. Another option here might be to give more options for the user to enter to customize place-items, or break it out into justify-content and align-items.

Reply:
yeah I guess you are right, we could always break this out into two props